tez/patches.yaml

```diff
@@ -0,0 +1,5 @@
+patches:
+# CVE-2024-7254
+- groupID: com.google.protobuf
+  artifactID: protobuf-java
+  version: 3.25.5
```

Comment:
Please note that while it bumps the `com.google.protobuf.protobuf-java` version from 3.24.4 to 3.25.5, the CVE is still not fixed as there's still a protobuf Hadoop-shaded dependency.

Reply:
Should we consider adding an advisory for it then?

Reply:
Yes, exactly.